jobs/integration/validation.py

```diff
@@ -2617,7 +2645,8 @@ async def test_cinder(model, tools):
 
 
 @pytest.mark.clouds(["openstack"])
-async def test_octavia(model, tools, teardown_microbot):
+@pytest.mark.usefixtures("teardown_microbot")
+async def test_octavia(model, tools):
```

Comment:
```suggestion
@pytest.mark.usefixtures("teardown_microbot")
async def test_octavia(model, tools):
# TODO: Update the microbot deployment mechanism since the action has been deprecated.
```